service/controller/key/key.go

```diff
@@ -39,10 +40,23 @@ func AppReferenced(app applicationv1alpha1.App, components map[string]releasev1a
 	return false
 }
 
+func ConfigReferenced(config corev1alpha1.Config, components map[string]releasev1alpha1.ReleaseSpecComponent) bool {
+	component, ok := components[config.Name]
+	if ok && IsSameConfig(component, config) {
+		return true
+	}
+
+	return false
+}
+
 func BuildAppName(component releasev1alpha1.ReleaseSpecComponent) string {
 	return fmt.Sprintf("%s-%s", component.Name, component.Version)
 }
 
+func BuildConfigName(component releasev1alpha1.ReleaseSpecComponent) string {
+	return fmt.Sprintf("%s-%s", component.Name, component.Version)
```

Comment:
Should we add `release-operator` to the name somehow? Otherwise I could see issues with opsctl / release-operator interfering. Not sure though.

Reply:
Isn't this the same for apps? Config name matches app name. I don't see where this can interfere

Reply:
If we stick to that convention in all other places, then yes this should be fine. `opsctl` creates apps usually with the branch name in the app name. 

But you are probably right and I am overthinking here. Just thought it might be beneficial to be more safe.